cmd/operator-sdk/generate/internal/genutil.go

```diff
@@ -79,7 +79,7 @@ func WriteCRDs(w io.Writer, crds ...v1beta1.CustomResourceDefinition) error {
 // WriteCRDFiles creates dir then writes each CustomResourceDefinition in crds
 // to a file in dir.
 func WriteCRDFiles(dir string, crds ...v1beta1.CustomResourceDefinition) error {
-	if err := os.MkdirAll(dir, 0700); err != nil {
+	if err := os.MkdirAll(dir, os.ModePerm); err != nil {
```

Comment:
It was before `0700` -> permission to rwx just for the owner
Now with this change, it will be `0777` -. permission to rwx owner, group and user. 

Since these filles are the manifests then, shows make sense.